src/job_metadata.c

```diff
@@ -982,7 +982,15 @@ TupleToCronJob(TupleDesc tupleDescriptor, HeapTuple heapTuple)
 									 tupleDescriptor, &isJobNameNull);
 		if (!isJobNameNull)
 		{
-			job->jobName = TextDatumGetCString(jobName);
+			/* Handle the column type change introduced in 1.5 */
+			if (TupleDescAttr(tupleDescriptor, Anum_cron_job_jobname - 1)->atttypid == NAMEOID)
+			{
+				job->jobName = (DatumGetName(jobName))->data;
```

Comment:
would you mind changing this to `pstrdup(NameStr(*DatumGetName(datum)));`?

need to copy the bytes here, and using `NameStr` is the usual convention

Reply:
oh, sure. thank you!